src/object/schedcontrol.c

```diff
@@ -115,23 +117,23 @@ static exception_t decodeSchedControl_Configure(word_t length, cap_t cap, extra_
         return EXCEPTION_SYSCALL_ERROR;
     }
 
-    if (budget_us > getMaxUsToTicks() || budget_us < MIN_BUDGET_US) {
+    if (budget > getMaxUsToTicks() || budget < MIN_BUDGET) {
```

Comment:
I did the conversion after as the checks are supposed to stop `usToTicks` from overflowing.

Reply:
I'll have another look at this. It appears I was mistaken when I first tried to resolve this.

The issue is that `usToTicks` depends on `div64` in some cases and I think there were issues with proving that. I'll talk to Miki some more. It sounds like if  change _does_ need to be made, this isn't the right one.